src/scripts/main.js

```diff
@@ -1,3 +1,65 @@
 'use strict';
 
-// write code here
+const queryList = document.querySelector('ul').children;
+
+const getEmployees = function(list) {
+  const arrayList = [...list].map(item => {
+    const itemEmployee = {
+      name: item.innerHTML.trim(),
+      position: item.getAttribute('data-position'),
+      salary: item.getAttribute('data-salary'),
+      age: item.getAttribute('data-age'),
```

Comment:
```suggestion
      ...item.dataset,
```

Reply:
this should work too

Reply:
then just position,  salary  and age will be stored inside dataset object

Reply:
fixed, thanks!